virttest/utils_test/qemu/__init__.py

```diff
@@ -186,6 +186,38 @@ def clear_win_driver_verifier(driver, vm, timeout=300):
         session.close()
 
 
+@error_context.context_aware
+def check_win_driver(test, driver, vm, timeout=300):
+    """
+    Check if driver is installed and state is running for windows guest.
+    :param test: QEMU test object.
+    :param driver: The driver which needs enable the verifier.
+    :param vm: VM object.
+    :param timeout: Timeout in seconds.
+    """
+    session = vm.wait_for_login(timeout=timeout)
+    logging.info("Sleep a while before check guest msinfo.")
+    time.sleep(10)
+    try:
+        error_context.context("Check %s driver state." % driver,
+                              logging.info)
+        msinfo_cmd = "start /wait msinfo32 /report c:\msinfo.txt"
+        driver_check_cmd = "type c:\msinfo.txt |findstr /i \"%s.sys\"" % driver
+
+        if session.cmd_status(msinfo_cmd, timeout=360):
+            test.error("Get msinfo32 failed.")
+
+        s_driver, o_driver = session.cmd_status_output(driver_check_cmd)
+        if s_driver or "running" in o_driver.lower():
+            msg = "%s driver is not installed or state is not running,the " \
+                  "detailed info is %s." % (driver, o_driver)
+            test.fail(msg)
+
+        logging.info("%s driver is running." % driver)
+    finally:
+        session.close
+
```

Comment:
should be session.close ()

Reply:
It's a typo when I update to github. Thanks for reminder.